### PR TITLE
Adding Elevation script + old shaped script releases

### PR DESCRIPTION
**Pull request by symposion**
Bringing my repo up-to-date with the roll20 one and submitting the Elevation change script for people to use

### CODE REVIEW

**Comment by Solarswordsman** [comment]
At some point your branch seems to have diverged pretty significantly from the main branch.  I'd recommend abandoning this branch and forking again so that you can just commit the script you'd like to add.

As it stands, this PR is going to modify far too many files to review.  Please resubmit with only new commits, based on this repository's master branch.

Thanks,
-Jeff

**Comment by Solarswordsman** [comment]
(Note: If your actual intention is to provide all of these versions of the shaped script I don't particularly mind, just please clarify with me if that's the case!  I can certainly understand why you'd want to do that.)

**Comment by symposion** [comment]
Hey,

It's your call, I guess I was just going for the easy option! I can't see much harm in it, but then again, I guess it's also slightly pointless having all those versions there. A while back I asked to remove the script from 1-click - not sure if that ever happened but we certainly stopped pushing releases there.

I can't see us wanting to put the Shaped Sheet into 1-click again any time soon. It's a versioning nightmare because there's no way to express a version dependency between the sheet and the script. We end up with people doing things like installing the script through 1 click but the sheet from github. They then get auto-updates on the script but not the sheet, and then complain when everything breaks out of the blue because the newly updated script requires a more recent version of the sheet; and then they download the most recent sheet and say they hate the changes; and then they tell us that they've been doing all this on their live game and is there any way to roll-back all the data updates RIGHT NOW because they've got a game in 5 minutes and oh my god we've ruined their lives :-)

TL;DR: I thought I might as well send you the old releases for completeness' sake, but there isn't a huge amount of practical value in it since Kryx is now hosting new releases of the script on another github fork that he's not pushing back to you guys for the reasons outlined above. If you want me to delete my fork and start again I can do that; but if you're not that bothered then it will save me messing around with git history if I can just submit as-is!

**Comment by Solarswordsman** [comment]
OK!  I will merge this in for completeness and availability on the repository.  The shaped companion script is set to hidden on the one-click, so users who already have it added could ostensibly switch versions, but no one can add it.

And I'll add the Elevation script to 1-click.

-Jeff
